src/main/java/oc/wh40k/units/or/ORWarboss.java

```diff
@@ -26,7 +26,7 @@ public ORWarboss() {
         waffen = new RuestkammerStarter(ID, randAbstand, cnt, "ORWaffenUndGeschenke", "");
         ((ORWaffenUndGeschenke)waffen.getKammer()).setDefaultFK("Kustom shoota");
         ((ORWaffenUndGeschenke)waffen.getKammer()).setDefaultNK("Big choppa");
-        waffen.initKammer(true,true,false,false,true);
+        waffen.initKammer(true,true,false,false,true,false,true);
```

Comment:
Could we please use something else then random booleans here? maybe an `EnumSet`?

Reply:
I'll add it as new task as this should be changed for the RuestkammerStarter in general then.

Reply:
Ok, Then it's ok for now I guess.